.github/workflows/oppiabot.yml

```diff
@@ -4,13 +4,22 @@ on:
   issues:
     types:
       - labeled
+  pull_request_target:
+    branches:
+      - develop
+      - release-*
 
 jobs:
   oppiabot:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: Ensure that appropriate labels are added to issues
```

Comment:
I'm not sure what to rename this as. How does "Github Actions from Oppiabot" sound?

Reply:
That sounds good.